als-node-client/node-package/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aml-org/als-node-client",
-  "version": "5.0.0-2",
+  "version": "5.1.0-SNAPSHOT",
```

Comment:
this shouldn't be a final version?

Reply:
package.json versions are not used, the version that matters is in dependency.properties

Reply:
5.0.0-2 was "used" because the hotfix was triggered manually I think @pope1838 right?